sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -3548,17 +3548,14 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase with Compilat
   }
 
   def failedToParseExistenceDefaultAsLiteral(fieldName: String, defaultValue: String): Throwable = {
-    new AnalysisException(
-      errorClass = "_LEGACY_ERROR_TEMP_1344",
-      messageParameters = Map(
-        "fieldName" -> fieldName,
-        "defaultValue" -> defaultValue))
+    SparkException.internalError(s"parse existence default as literal err," +
+      s" field name: $fieldName, value: $defaultValue")
   }
 
   def defaultReferencesNotAllowedInDataSource(
       statementType: String, dataSource: String): Throwable = {
     new AnalysisException(
-      errorClass = "_LEGACY_ERROR_TEMP_1345",
+      errorClass = "DEFAULT_UNSUPPORTED",
       messageParameters = Map(
         "statementType" -> statementType,
```

Comment:
If it is a SQL statement, quote it by `toSQLStmt()`, please.

Reply:
According to the function signature and current callers, value passed to param statementType is not necessarily a grammatically correct SQL statement(like 'ALTER TABLE', 'CREATE TABLE'), and it seems no need to quote it by toSQLStmt() here ?

Reply:
> If it is a SQL statement, quote it by `toSQLStmt()`, please.

According to the function signature and current callers, value passed to param statementType is not necessarily a grammatically correct SQL statement(like 'ALTER TABLE', 'CREATE TABLE'), and it seems no need to quote it by toSQLStmt() here ?

Reply:
> grammatically correct SQL statement

It shouldn't be a correct SQL statement. If it is a part of SQL statements, please, use `toSQLStmt()`

Reply:
Ok, I will modify it together after @dtenedor confirm the change 44 to internal err.